packages/service-library/tests/test_pools.py

```diff
@@ -0,0 +1,34 @@
+from asyncio import BaseEventLoop
+
+from servicelib.pools import non_blocking_process_pool_executor
+from concurrent.futures import ProcessPoolExecutor
+import pytest
+from concurrent.futures import Executor
+
```

Comment:
I find this tests extremely verbose for such a simple thing. Some tips:

- you can avoid all these decorators by marking the entire file just once:
```python
import pytest

pytestmark = pytest.mark.asyncio
```
- Name the dummy function to indicate its role in the test. In this ase is to provide a synchronous function that you are going to execute as "async"
- There is really no need to return in tests ... ``-> None ``. It does not add any info and obfuscates
- Why to  create so many indirections for something so simple? Just using a single like
``assert await event_loop.run_in_executor(executor, return_int_one) == 1 `` is extremely clearly stated the intention. No need to add another inderection

In conclusion, I would have written it as simple as follows. 

```python
import pytest
import time

pytestmark = pytest.mark.asyncio

def sync_fun():
   time.sleep(.1)
   return 1

async def test_default_thread_pool_executor(event_loop: BaseEventLoop):
    assert await event_loop.run_in_executor(None, sync_fun) == 1 

async def test_blocking_process_pool_executor(event_loop: BaseEventLoop):
    with ProcessPoolExecutor() as executor:
        assert await event_loop.run_in_executor(executor, sync_fun) == 1 

async def test_non_blocking_process_pool_executor(event_loop: BaseEventLoop):
    with non_blocking_process_pool_executor() as executor:
        assert await event_loop.run_in_executor(executor, sync_fun) == 1 
```